rest-api/src/main/java/com/cloudbees/workflow/flownode/FlowNodeUtil.java

```diff
@@ -249,12 +384,17 @@ public static FlowNode getStageEndNode(FlowNode stageStartNode) {
         return allNodesSorted.get(allNodesSorted.size() - 1);
     }
 
+    @CheckForNull
```

Comment:
Thanks, noted, +1 on this, if this gets another commit (beyond merges from master), I will move it, otherwise, will do in the other PR.